vcmibuilder

```diff
@@ -236,12 +227,6 @@ then
 	cp -r "$data_dir"/[Mm][Pp]3 "$dest_dir"
 fi
 
-if [[ -n "$download" ]]
-then
-	wget "http://download.vcmi.eu/core.zip" -O "$temp_dir"/core.zip || fail "Error: failed to download VCMI archive!" "rm -f core.zip"
-	vcmi_archive="$temp_dir"/core.zip
-fi
-
 if [[ -n "$vcmi_archive" ]]
 then
 	echo "decompressing $vcmi_archive"
```

Comment:
please delete the `vcmi_archive` part as well as it's directly related to `download`